cypress/page/address-step.page.ts

```diff
@@ -2,7 +2,9 @@ class AddressStepPage {
     private proceedToCheckoutBtn: string;
 
     constructor() {
-        this.proceedToCheckoutBtn = ".cart_navigation > .button > span";
+        // Eliminate span, If the purpose is to click a button its unnecesary to reach for the span element, we can just eliminate this extra step and click the container
+        // also added an additional argument to match the name of the button because i dont think the class button is a good enough identifier and it is not scalable
```

Comment:
Thanks for the clarification, its better the code its left uncommented, you can add the info on the PR

But really thank you for the explanation